CHANGELOG.md

```diff
@@ -3,19 +3,34 @@ Write the date in place of the "Unreleased" in the case a new version is release
 
 # Changelog
 
-## Unreleased
+## v0.1.0b8 (2024-09-05)
+
+### Fixed
+
+- Compatibility with a change in Starlette v0.38.0
+
+## v0.1.0b7 (2024-08-20)
 
 ### Added
+
 - Add method to `TableAdapter` which accepts a Python dictionary.
 - Added an `Arrow` adapter which supports reading/writing arrow tables via `RecordBatchFileReader`/`RecordBatchFileWriter`.
 
 ### Changed
+
 - Make `tiled.client` accept a Python dictionary when fed to `write_dataframe()`.
 - The `generated_minimal` example no longer requires pandas and instead uses a Python dict.
 - Remove unused pytest-warning ignores from `test_writing.py`.
 - Rename argument in `hdf5_lookup` function from `path` to `dataset` to reflect change in `ophyd_async`
 
+## v0.1.0b6 (2024-07-17)
```

Comment:
Releases for b6 and b7 were made on GH, but the CHANGELOG was not updated.

The release b8 will go out today (I intend) to release the fixes in this PR.